src/pytest_workflow/plugin.py

```diff
@@ -52,6 +58,28 @@ def pytest_collect_file(path, parent):
     return None
 
 
+def pytest_configure(config: Config):
+    """This runs before tests start and adds values to the config."""
+    # We need to add a workflow queue to some central variable. Instead of
+    # using a global variable we add a value to the config.
+    # Configuring was copied from _pytest/tmpdir.py. There they also add
+    # objects to the config.
+    # pylint: disable=protected-access
+    # protected access is needed here to do it the pytest way: ugly.
+    monkeypatch = MonkeyPatch()
+    config._cleanup.append(monkeypatch.undo)
+    workflow_queue = WorkflowQueue()
+    monkeypatch.setattr(config, "workflow_queue", workflow_queue,
```

Comment:
I know this is pytest's internals showing its guts, but man, this should make everyone weep. 
Does python's builtin `setattr` not work here? I.e. simply `setattr(config, 'workflow_queue', workflow_queue)` ?